_includes/tutorials/ksql-heterogeneous-json/ksql/markup/dev/problem-description.adoc

```diff
@@ -0,0 +1,34 @@
+Let's say you have a topic `source_data`, that contains JSON.  But each nested JSON object has a different structure.  Addtionally, in each object there is a mix of types within the values.
+
+They each have one field in common that you want to pull out in a query and you don't care about the structure of the individual JSON objects
+
+[source, json]
+----
+{
+  "JSONTypeOne": {
+    "fieldA": "some data",
+    "numberField": 1.001,
+    "oneOnlyField": "more data", <1>
+    "randomField": "random data"
+  },
+
+  "JSONTypeTwo": {
+    "fieldA": "data",
+    "fieldB": "b-data",
+    "numberField": 98.6   <2>
+  },
+
+  "JSONTypeThree": {
```

Comment:
In my environment: I can't get this this third record to display in Safari unless I expand.  Scrolling does not help.

Can you please try in your view: do you see this 3rd record type?

Reply:
No, I have to expand as well.  It's like this with all KT's, maybe we need to explore the default of having it expand or allow scrolling.

Reply:
As a workaround, what do you think about splitting each of the 3 JSON types into their own `[source, json]` code block?

Reply:
ack